modules/sync/syncs/httpsynchronization.cpp

```diff
@@ -134,7 +163,85 @@ std::string HttpSynchronization::generateUid() {
     return fmt::format("{}/{}", _identifier, _version);
 }
 
-bool HttpSynchronization::trySyncFromUrl(std::string listUrl) {
+void HttpSynchronization::createSyncFile(bool isFullySynchronized) const {
+    std::filesystem::path dir = directory();
+    std::filesystem::create_directories(dir);
+
+    dir.replace_extension("ossync");
+    std::ofstream syncFile(dir, std::ofstream::out);
+
+    syncFile << _ossyncVersionNumber << '\n' <<
+        (isFullySynchronized ? _synchronizationToken : "Partial Synchronized") << '\n';
```

Comment:
Maybe easier to read?

```suggestion
    syncFile << fmt::format(
        "{}\n{}\n",
        _ossyncVersionNumber,
        (isFullySynchronized ? _synchronizationToken : "Partial Synchronized")
    );
```